### PR TITLE
Update docs with default admin credentials to explain init

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
5.16 adds `sensu-backend init`, which removes default admin username and password when installing the Sensu backend. This PR updates all other docs that mentioned the previous default password (P@ssw0rd!) to explain that the admin username and password were set up during the initialization step when installing the Sensu backend.

## Motivation and Context
Need to carry changes based on https://github.com/sensu/sensu-docs/pull/1962 through all docs.

## Review Instructions
Please confirm that my explanations are complete -- I'm not sure if I've provided enough information in some places.

### CODE REVIEW

File: content/sensu-go/5.16/getting-started/prometheus-metrics.md [hunk 1]
```diff
@@ -372,7 +372,7 @@ sensuctl create --file check --file asset_prometheus
 {{< /highlight >}}
 
 We can see the events generated by the `prometheus_metrics` check in the Sensu dashboard.
-Visit http://127.0.0.1:3000, and log in as the default admin user: username `admin` and password `P@ssw0rd!`.
+Visit http://127.0.0.1:3000, and log in as the default admin user (created during the [initialization step][8] when you installed the Sensu backend).
```
Comment (cwjohnston) [inline content/sensu-go/5.16/getting-started/prometheus-metrics.md]:
```suggestion
Visit http://127.0.0.1:3000, and log in as the admin user (created during the [initialization step][8] when you installed the Sensu backend).
```

File: content/sensu-go/5.16/api/overview.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ The [`/auth` API endpoint][9] lets you generate short-lived API tokens using you
 1. Retrieve an access token for your user.
 For example, to generate an access token using the default admin credentials:
```
Comment (cwjohnston) [inline content/sensu-go/5.16/api/overview.md]:
```suggestion
For example, to generate an access token using example admin credentials:
```

File: content/sensu-go/5.16/files/agent.yml [hunk 3]
```diff
@@ -36,7 +36,7 @@ log-level: "debug" # available log levels: panic, fatal, error, warn, info, debu
 # authentication configuration
 ##
 #user: "agent"
-#password: "P@ssw0rd!"
+#password: "AGENT_PASSWORD"
```
Comment (cwjohnston) [inline content/sensu-go/5.16/files/agent.yml]: From https://github.com/sensu/sensu-go/blob/release/5.16/backend/seeds/seeds.go#L76-L79 I'm understanding that the agent default password remains intact for this release.

```suggestion
#password: "P@ssw0rd!"
```

File: content/sensu-go/5.16/files/windows/agent.yml [hunk 4]
```diff
@@ -35,7 +35,7 @@ log-level: "debug" # available log levels: panic, fatal, error, warn, info, debu
 # authentication configuration
 ##
 #user: "agent"
-#password: "P@ssw0rd!"
+#password: "AGENT_PASSWORD"
```
Comment (cwjohnston) [inline content/sensu-go/5.16/files/windows/agent.yml]:
```suggestion
#password: "P@ssw0rd!"
```

File: content/sensu-go/5.16/reference/agent.md [hunk 5]
```diff
@@ -717,7 +717,7 @@ Flags:
       --log-level string                logging level [panic, fatal, error, warn, info, debug] (default "warn")
       --name string                     agent name (defaults to hostname) (default "sensu-go-sandbox")
       --namespace string                agent namespace (default "default")
-      --password string                 agent password (default "P@ssw0rd!")
+      --password string                 agent password (set up during [initialization step][50] in Sensu backend installation)
```
Comment (cwjohnston) [inline content/sensu-go/5.16/reference/agent.md]:
```suggestion
      --password string                 agent password (default "P@ssw0rd!")
```

File: content/sensu-go/5.16/reference/agent.md [hunk 6]
```diff
@@ -1026,12 +1026,12 @@ user: "agent-01"{{< /highlight >}}
 --------------|------
 description   | Sensu [RBAC](../rbac) password used by the agent
 type          | String
-default       | `P@ssw0rd!`
+default       | `RBAC_PASSWORD`
```
Comment (cwjohnston) [inline content/sensu-go/5.16/reference/agent.md]:
```suggestion
default       | `P@ssw0rd!`
```

File: content/sensu-go/5.16/reference/agent.md [hunk 7]
```diff
@@ -1026,12 +1026,12 @@ user: "agent-01"{{< /highlight >}}
...
 example       | {{< highlight shell >}}# Command line example
-sensu-agent start --password secure-password
+sensu-agent start --password RBAC_PASSWORD
```
Comment (cwjohnston) [inline content/sensu-go/5.16/reference/agent.md]:
```suggestion
sensu-agent start --password secure-password
```

File: content/sensu-go/5.16/reference/agent.md [hunk 8]
```diff
@@ -1026,12 +1026,12 @@ user: "agent-01"{{< /highlight >}}
...
 
 # /etc/sensu/agent.yml example
-password: "secure-password"{{< /highlight >}}
+password: "RBAC_PASSWORD"{{< /highlight >}}
```
Comment (cwjohnston) [inline content/sensu-go/5.16/reference/agent.md]:
```suggestion
password: "secure-password"{{< /highlight >}}
```

File: content/sensu-go/5.16/reference/agent.md [hunk 9]
```diff
@@ -1339,3 +1339,4 @@ $ sudo systemctl restart sensu-agent
 [47]: https://en.m.wikipedia.org/wiki/Protocol_Buffers
 [48]: #example-allow-list-configuration-file
 [49]: #allow-list-configuration
+[50]: ../../installation/install-sensu/#2-initialize
```
Comment (cwjohnston) [inline content/sensu-go/5.16/reference/agent.md]:
```suggestion
```

File: content/sensu-go/5.16/getting-started/sample-app.md [hunk 10]
```diff
@@ -112,13 +112,15 @@ kubectl create -f go/deploy/sensu-backend.yaml
 
 Use Sensu role-based access control to create a `demo` namespace and a `demo` user.
 
-**1. Configure sensuctl to use the built-in `admin` user.**
+**1. Configure sensuctl to use the admin user.**
```
Comment (cwjohnston) [inline content/sensu-go/5.16/getting-started/sample-app.md:115]: I think the changes here work for now, but we'll likely need another update to this document once https://github.com/sensu/sensu-kube-demo/issues/25 is implemented.

File: content/sensu-go/5.16/installation/install-sensu.md [hunk 11]
```diff
@@ -220,8 +220,8 @@ To start using sensuctl, run `sensuctl configure` and log in with your user cred
 
 {{< highlight "shell" >}}
 sensuctl configure -n \
---username 'admin' \
---password 'P@ssw0rd!' \
+--username 'YOUR_USERNAME' \
+--password 'YOUR_PASSWORD' \
```
Comment (cwjohnston) [inline content/sensu-go/5.16/installation/install-sensu.md:236]: This works, but I think we also need to update docker installation commands around line 101 (docker) and 108 (docker compose) to include init step.

File: content/sensu-go/5.16/reference/rbac.md [hunk 12]
```diff
@@ -219,12 +219,14 @@ You can use your Sensu username and password to [configure sensuctl][26] or log
 
 ### Default user
 
-By default, Sensu includes a global `admin` user that you can use to manage Sensu and create new users.
+When you install the Sensu backend, during the [initialization step][41], you create a username and password for a `default` namespace.
+
+This is the global user that you can use to manage Sensu and create new users.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/reference/rbac.md]:
```suggestion
This is the user that you can use to manage all aspects of Sensu and create new users.
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]

File: content/sensu-go/5.16/installation/install-sensu.md (hunk 11 above)
Comment (hillaryfraley) [reply content/sensu-go/5.16/installation/install-sensu.md:236]: @cwjohnston I'm not totally sure how to do that -- Should there be separate sections for Docker in the Initialize section (like at line 91)?
Or would they need to run sensu-backend init again for every Docker container?

Comment (hillaryfraley) [reply content/sensu-go/5.16/installation/install-sensu.md:236]: @rgeniesse pointed out that current plans call for keeping Docker as-is (with default admin/P@ssw0rd!) for 5.16 and rolling out `sensu-backend init` for Docker in 5.17. See https://sensu.slack.com/archives/C60EEQFH8/p1576190537067300 for context.

File: content/sensu-go/5.16/installation/install-sensu.md [hunk 13]
```diff
@@ -85,6 +85,8 @@ sudo yum install sensu-go-backend
 
 ### 2. Initialize
 
+_**NOTE**: If you are using Docker, skip this step and continue with [3. Configure and start][36]. The `sensu-backend init` command is not implemented for Docker._
+
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/installation/install-sensu.md:89]: I added this note based on https://sensu.slack.com/archives/C60EEQFH8/p1576190537067300

If this note is correct, I can add it in the other docs in this PR as appropriate.

File: content/sensu-go/5.16/getting-started/sample-app.md [hunk 14]
```diff
@@ -112,13 +112,15 @@ kubectl create -f go/deploy/sensu-backend.yaml
...
+
+When you install the Sensu backend, during the [initialization step][40], you created an admin username and password for a `default` namespace. Use that username and password to configure sensuctl in this step.
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/getting-started/sample-app.md]:
```suggestion
When you installed the Sensu backend, during the [initialization step][40], you created an admin username and password for a `default` namespace. Use that username and password to configure sensuctl in this step.
```

**Comment by hillaryfraley** [comment]
When we're ready to add the Docker commands, see https://github.com/sensu/website/pull/473#issuecomment-566238246

File: content/sensu-go/5.16/installation/install-sensu.md (hunk 13 above)
Comment (rgeniesse) [reply content/sensu-go/5.16/installation/install-sensu.md:89]: Maybe: "The `sensu-backend init` command is automatically ran for you with the default username and password of `admin` and `P@ssw0rd!`."

Saying "The `sensu-backend init` command is not implemented for Docker." is incorrect as the binary inside the container will have that subcommand available, you just don't need to run it to get the admin user provisioned as we are taking care of that for you in 5.16. And then you shouldn't need it again, as you would then just use `sensuctl configure` as the admin user and do all your normal stuff from there.

File: content/sensu-go/5.16/installation/install-sensu.md (hunk 11 above)
Comment (rgeniesse) [reply content/sensu-go/5.16/installation/install-sensu.md:236]: Yeah, so based on that we won't need to init the docker container; it'll be done for the users in the building of the image. So for those users, they won't notice a difference. But then in 5.17, they'll need to do it. At least, that's the plan.

File: content/sensu-go/5.16/installation/install-sensu.md [hunk 15]
```diff
@@ -85,6 +85,8 @@ sudo yum install sensu-go-backend
 
 ### 2. Initialize
 
+_**NOTE**: If you are using Docker, skip this step and continue with [3. Configure and start][36]. The `sensu-backend init` command is not implemented for Docker._
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/installation/install-sensu.md:88]:
```suggestion
_**NOTE**: If you are using Docker, skip this step and continue with [3. Configure and start][36]. The `sensu-backend init` command runs automatically with a default username (`admin`) and password (`P@ssw0rd!`) for Docker._
```

File: content/sensu-go/5.16/installation/install-sensu.md (hunk 13 above)
Comment (hillaryfraley) [reply content/sensu-go/5.16/installation/install-sensu.md:89]: ^ updated @rgeniesse wdyt?

**Review by palourde — APPROVED** [review]
👍

File: content/sensu-go/5.16/reference/rbac.md [hunk 16]
```diff
@@ -219,12 +219,14 @@ You can use your Sensu username and password to [configure sensuctl][26] or log
 
 ### Default user
 
-By default, Sensu includes a global `admin` user that you can use to manage Sensu and create new users.
+When you install the Sensu backend, during the [initialization step][41], you create a username and password for a `default` namespace.
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/reference/rbac.md:222]:
```suggestion
When you install the Sensu backend, during the [initialization step][41], you create a username and password for a `default` namespace.

_**NOTE**: If you are using Docker, the `sensu-backend init` command runs automatically with a default username (`admin`) and password (`P@ssw0rd!`). You do not need to create a username and password for the `default` namespace if you are using Docker._
```

**Pull request merged**